dotcom-rendering/src/components/ImageComponent.tsx

```diff
@@ -353,13 +378,28 @@ export const ImageComponent = ({
 				{!!title && (
 					<ImageTitle title={title} role={role} palette={palette} />
 				)}
+				{parseInt(imageWidth, 10) > 620 &&
+					element.position !== undefined && (
+						<LightboxLink
+							role={role}
+							format={format}
+							elementId={element.elementId}
+							isMainMedia={isMainMedia}
+							position={element.position}
+						/>
+					)}
 			</div>
 		);
 	}
 
 	return (
 		<>
 			<div
+				id={
+					element.position !== undefined
+						? `img-${element.position}`
+						: ''
```

Comment:
This should be `undefined` so the attribute is removed entirely. Cannot find this behaviour explicitely documentend in the [new React docs](https://react.dev/reference/react-dom/components/common), but I think it’s correct.